### PR TITLE
Add of documentation

### PR DESCRIPTION
**Pull request by Baribal02**
Add of documentation to most of the functions that we have
Commented and unused code has also been supressed

### CODE REVIEW

File: app/src/main/java/ch/epfl/skysync/components/FlightsList.kt [hunk 1]
```diff
@@ -61,10 +78,15 @@ fun TopBanner(topTitle: String, topBannerColor: Color, paddingValues: PaddingVal
 }
 
 /**
- * shows a list of flights. Distinguishes 3 cases:
+ * A composable function that displays the content of the flights list. It distinguishes 3 cases:
  * 1) flights == null: shows a loading component
  * 2) flights is empty: shows a text "No flights"
  * 3) flights not empty: shows a the of flights
```
Comment (Prolas) [inline app/src/main/java/ch/epfl/skysync/components/FlightsList.kt:84]: It's not yours but a part is missing

File: app/src/main/java/ch/epfl/skysync/models/calendar/AvailabilityStatus.kt [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
- * Represent three statuses: "OK" for available, "MAYBE" for pending confirmation, and "NO" for not
- * available
+ * Enum class representing the availability status of a person.
+ *
+ * The status can be one of the following:
+ * - OK: The person is available.
+ * - MAYBE: The person's availability is pending confirmation.
+ * - NO: The person is not available.
+ * - UNDEFINED: The availability status is not defined.
+ * - ASSIGNED: The person is assigned to a task.
```
Comment (Prolas) [inline app/src/main/java/ch/epfl/skysync/models/calendar/AvailabilityStatus.kt:11]: I would prefer: The person is assigned to a flight.

File: app/src/main/java/ch/epfl/skysync/models/calendar/FlightGroupCalendar.kt [hunk 3]
```diff
@@ -33,12 +52,13 @@ class FlightGroupCalendar(cells: List<FlightGroup> = listOf()) : CalendarModel<F
       setByDate(date, timeSlot) { d, t, old -> flightGroup } as FlightGroupCalendar
 
   /**
-   * adds the given flight to the existing FlightGroup at slot for (date, timeSlot) or adds a new
-   * FlightGroup if non exists
+   * Adds the given flight to the existing FlightGroup at slot for (date, timeSlot) or adds a new
+   * FlightGroup if non exists.
```
Comment (Prolas) [inline app/src/main/java/ch/epfl/skysync/models/calendar/FlightGroupCalendar.kt:56]: Nitpick: if doesn't exist

File: app/src/main/java/ch/epfl/skysync/models/user/Crew.kt [hunk 4]
```diff
@@ -3,6 +3,15 @@ package ch.epfl.skysync.models.user
 import ch.epfl.skysync.models.UNSET_ID
 import ch.epfl.skysync.models.flight.RoleType
 
+/**
+ * Represents an admin user
```
Comment (Prolas) [inline app/src/main/java/ch/epfl/skysync/models/user/Crew.kt:7]: I think it is: Represents a crew user

File: app/src/main/java/ch/epfl/skysync/screens/crewpilot/Calendar.kt [hunk 5]
```diff
@@ -78,6 +86,13 @@ fun CalendarScreen(
       }
 }
 
+/**
+ * The top bar for the calendar
+ *
+ * @param tab The current tab
+ * @param tabs The tabs
+ * @param onclick The onclick function
```
Comment (Prolas) [inline app/src/main/java/ch/epfl/skysync/screens/crewpilot/Calendar.kt:94]: The function to switch between the tabs

File: app/src/main/java/ch/epfl/skysync/screens/crewpilot/Flight.kt [hunk 6]
```diff
@@ -57,6 +57,12 @@ import com.google.maps.android.compose.MarkerState
...
 import com.google.maps.android.compose.rememberCameraPositionState
 
+/**
+ * This composable function renders a marker on the map at the given location.
+ *
+ * @param location The location to render the marker at.
+ * @param user The user associated with the location.
+ */
 @Composable
 fun UserLocationMarker(location: Location, user: User) {
```
Comment (Prolas) [inline app/src/main/java/ch/epfl/skysync/screens/crewpilot/Flight.kt:67]: This function is a duplication of the function in `app/src/main/java/ch/epfl/skysync/screens/admin/AdminFlight.kt`

**Review by Prolas — APPROVED** [review]
The params are missing in 
* app/src/main/java/ch/epfl/skysync/util/FormsUtil.kt
* app/src/main/java/ch/epfl/skysync/viewmodel/ChatViewModel.kt
* app/src/main/java/ch/epfl/skysync/viewmodel/FinishedFlightsViewModel.kt
* app/src/main/java/ch/epfl/skysync/viewmodel/FlightsViewModel.kt
* app/src/main/java/ch/epfl/skysync/viewmodel/InFlightViewModel.kt
* app/src/main/java/ch/epfl/skysync/viewmodel/UserGlobalViewModel.kt

File: app/src/main/java/ch/epfl/skysync/components/FlightsList.kt (hunk 1 above)
Comment (Baribal02) [reply app/src/main/java/ch/epfl/skysync/components/FlightsList.kt:84]: It is modified

File: app/src/main/java/ch/epfl/skysync/models/calendar/AvailabilityStatus.kt (hunk 2 above)
Comment (Baribal02) [reply app/src/main/java/ch/epfl/skysync/models/calendar/AvailabilityStatus.kt:11]: Yeah, I have changed that

File: app/src/main/java/ch/epfl/skysync/models/calendar/FlightGroupCalendar.kt (hunk 3 above)
Comment (Baribal02) [reply app/src/main/java/ch/epfl/skysync/models/calendar/FlightGroupCalendar.kt:56]: Corrected !

File: app/src/main/java/ch/epfl/skysync/models/user/Crew.kt (hunk 4 above)
Comment (Baribal02) [reply app/src/main/java/ch/epfl/skysync/models/user/Crew.kt:7]: Indead, it is now correct

File: app/src/main/java/ch/epfl/skysync/screens/crewpilot/Calendar.kt (hunk 5 above)
Comment (Baribal02) [reply app/src/main/java/ch/epfl/skysync/screens/crewpilot/Calendar.kt:94]: Changed it

File: app/src/main/java/ch/epfl/skysync/screens/crewpilot/Flight.kt (hunk 6 above)
Comment (Baribal02) [reply app/src/main/java/ch/epfl/skysync/screens/crewpilot/Flight.kt:67]: Yes, I'll let that to Rafael who's task is to clean the composables code.

**Pull request merged**